Post de presentación e incorporación a la coordinación

Nkrunsh opened the pull request:
Cambios Realizados:
- Incorporación de Imagen 300x300 px
- Creación de Post de Presentación
- Incorporación a Coordinación

Nkrunsh committed:
Post de presentación e incorporación a la coordinación

KrlitosForever commented on content/2025-04-20-presentacion-erickson-gallardo.md:
```diff
@@ -0,0 +1,84 @@
+## Introducción breve
+
+¡Hola, querida comunidad de Python Chile!  
+Mi nombre es Erickson Gallardo y tengo el gusto de presentarme como uno de los nuevos integrantes del equipo de coordinación. Actualmente vivo en Santiago, Chile, y me formé como Ingeniero Civil Informático.  
```
```suggestion
Mi nombre es Erickson Gallardo y tengo el gusto de presentarme como uno de los nuevos integrantes del equipo de coordinación. Actualmente vivo en Santiago de Chile y me formé como Ingeniero Civil Informático.  
```

KrlitosForever commented on content/2025-04-20-presentacion-erickson-gallardo.md:
```diff
@@ -0,0 +1,84 @@
+¡Hola, querida comunidad de Python Chile!  
+Mi nombre es Erickson Gallardo y tengo el gusto de presentarme como uno de los nuevos integrantes del equipo de coordinación. Actualmente vivo en Santiago, Chile, y me formé como Ingeniero Civil Informático.  
+
+Desde que egresé, he trabajado por más de cinco años promoviendo el uso de tecnologías en el aula de manera transversal, abarcando áreas como la programación, la electrónica, el diseño y la fabricación digital. Ha sido una experiencia increíble ver cómo la tecnología puede transformar la forma en que aprendemos y enseñamos.
```
```suggestion
Desde que egresé, he trabajado por más de cinco años promoviendo el uso de tecnologías en el aula de manera transversal, abarcando áreas como la programación, la electrónica, el diseño y la fabricación digital. Ha sido una experiencia increíble ver como la tecnología puede transformar la forma en que aprendemos y enseñamos.
```

KrlitosForever commented on content/2025-04-20-presentacion-erickson-gallardo.md:
```diff
@@ -0,0 +1,84 @@
+
+## ¿Cómo descubriste Python y qué te motivó a aprenderlo?
+
+Mi entrada al mundo de Python fue, curiosamente, bastante informal. No tengo un recuerdo exacto de haberme sentado a "aprenderlo", pero sí recuerdo cómo se fue metiendo poco a poco en mi vida.  
```
```suggestion
Mi entrada al mundo de Python fue, curiosamente, bastante informal. No tengo un recuerdo exacto de haberme sentado a "aprenderlo", pero sí recuerdo como se fue metiendo poco a poco en mi vida.  
```

Tony-Rome replied:
Entiendo que esta sugerencia no debería ir ya que "cómo" se ocupa como pronombre interrogativo.

KrlitosForever commented on content/2025-04-20-presentacion-erickson-gallardo.md:
```diff
@@ -0,0 +1,84 @@
+
+Durante mis años universitarios, tenía un amigo fanático de Python (¡y todavía lo es!). Siempre nos contaba con entusiasmo sobre sus proyectos y desafíos personales, experimentando con cosas nuevas solo por el gusto de aprender. Gracias a su pasión fui conociendo el lenguaje y todo el potencial que tenía.  
+
+Ya titulado, decidí profundizar en Python a través de cursos en línea, y comencé a aplicarlo en mi trabajo: desde proyectos escolares hasta clases de tecnología. Fue ahí donde realmente comence a desarrollar en este lenguaje.
```
```suggestion
Ya titulado, decidí profundizar en Python a través de cursos en línea, y comencé a aplicarlo en mi trabajo: desde proyectos escolares hasta clases de tecnología. Fue ahí donde realmente comencé a desarrollar en este lenguaje.
```

KrlitosForever commented on integrantes.py:
```diff
@@ -298,4 +298,15 @@
+        "github": "Nkrunsh",
+        "activo": "1",
+        "descripcion": (
+            "Ingeniero Civil Informático, apasionado por el desarrollo Maker y proyectos DIY."
```
```suggestion
            "Ingeniero Civil Informático, apasionado por el desarrollo Maker y proyectos DIY. "
```

KrlitosForever commented on integrantes.py:
```diff
@@ -298,4 +298,15 @@
+        "activo": "1",
+        "descripcion": (
+            "Ingeniero Civil Informático, apasionado por el desarrollo Maker y proyectos DIY."
+            "Más de 5 años de experiencia aplicando programación, electrónica y fabricación digital en el aula."
```
```suggestion
            "Más de 5 años de experiencia aplicando programación, electrónica y fabricación digital en el aula. "
```

KrlitosForever commented on content/2025-04-20-presentacion-erickson-gallardo.md:
```diff
@@ -0,0 +1,84 @@
+
+Lo más importante es **conocerse a uno mismo** en cómo aprendemos. Por ejemplo, yo tengo un estilo de aprendizaje muy visual, así que me sirven mucho los tutoriales con ejemplos prácticos.  
+
+Una vez que entiendo la estructura baisca y funcionamiento general de un código, empiezo a experimentar: cambio variables, agrego funciones, pruebo qué pasa si hago esto o aquello… Y claro, cuando aparecen errores (¡que siempre aparecen!), busco en internet o recurro a herramientas de IA. Hoy en día, los modelos de lenguaje como ChatGPT son un apoyo increíble para entender conceptos y resolver du ...
```
```suggestion
Una vez que entiendo la estructura básica y funcionamiento general de un código, empiezo a experimentar: cambio variables, agrego funciones, pruebo qué pasa si hago esto o aquello… y claro, cuando aparecen errores (¡que siempre aparecen!), busco en internet o recurro a herramientas de IA. Hoy en día, los modelos de lenguaje como ChatGPT son un apoyo increíble para entender conceptos y resolver dudas paso a paso.
```

KrlitosForever commented on content/2025-04-20-presentacion-erickson-gallardo.md:
```diff
@@ -0,0 +1,84 @@
+
+Python se ha convertido en mi herramienta favorita para automatizar tareas y desarrollar soluciones prácticas en el día a día.  
+
+Algunos de mis proyectos favoritos han sido:
```
```suggestion
Algunos de mis proyectos favoritos han sido:

```

KrlitosForever commented on content/2025-04-20-presentacion-erickson-gallardo.md:
```diff
@@ -0,0 +1,84 @@
+Python se ha convertido en mi herramienta favorita para automatizar tareas y desarrollar soluciones prácticas en el día a día.  
+
+Algunos de mis proyectos favoritos han sido:
+- Una aplicación con interfaz en Tkinter para recibir y ejecutar comandos desde dispositivos Micro:Bit, integrándolos con proyectos en Scratch.
```
```suggestion
- Una aplicación con interfaz en Tkinter para recibir y ejecutar comandos desde dispositivos Micro:Bit, integrándolos con proyectos en Scratch.

```

KrlitosForever commented on content/2025-04-20-presentacion-erickson-gallardo.md:
```diff
@@ -0,0 +1,84 @@
+
+Algunos de mis proyectos favoritos han sido:
+- Una aplicación con interfaz en Tkinter para recibir y ejecutar comandos desde dispositivos Micro:Bit, integrándolos con proyectos en Scratch.
+- Un pequeño sistema para automatizar el ingreso y limpieza de datos, útil en tareas administrativas.
```
```suggestion
- Un pequeño sistema para automatizar el ingreso y limpieza de datos, útil en tareas administrativas.

```

KrlitosForever reviewed (commented):
Excelente @Nkrunsh ahí te deje unos comentarios por si quieres aplicar los cambios sugeridos.
Una vez me comentes lo reviso nuevamente 👍🏻

Tony-Rome commented on content/images/coordinacion_erickson_gallardo.jpg:
Se pide en la [docu](https://github.com/python-chile/docs/blob/master/formato-presentacion-blog.md) que sea de 330 x 330 la imagen, aunque viendo el resto de imagenes parece ser que no es obligación.

Tony-Rome reviewed (changes requested):
Me sumo a las sugerencias de Carlos, son detalles pequeños. Lo que sí pido, favor actualizar nombre de la rama, cambiar la palabra *task*  por *change* o *feature*, por ejemplo *pch-feature-148-perfil-post* 

Por lo demás, gracias por la creación del post! 🧙‍♂️

KrlitosForever commented:
y no olvidarse de darle estrellita al repo ⭐

Nkrunsh committed:
Aplicación de sugerencias post de presentación

Se aplican los cambio sugeridos en primera revisión para contenido del post de presentación.

Co-authored-by: Carlos Carrasco Varas <matrix.carlos@gmail.com>
